packages/react-native-worklets/Common/cpp/worklets/WorkletRuntime/WorkletRuntime.h

```diff
@@ -69,7 +70,16 @@ class WorkletRuntime : public jsi::HostObject,
 
   std::vector<jsi::PropNameID> getPropertyNames(jsi::Runtime &rt) override;
 
+  [[nodiscard]] auto getRuntimeId() const -> uint64_t {
```

Comment:
As said previously, I'm not a fan of using `auto getRuntimeId() const -> uint64_t { ... }` in a codebase where we mostly use `uint64_t getRuntimeId() const { ... }`. Let's discuss it with the team.